.github/workflows/release.yaml

```diff
@@ -35,12 +35,14 @@ jobs:
           registry: "quay.io"
           username: "${{ secrets.QUAY_USERNAME }}"
           password: "${{ secrets.QUAY_ROBOT_TOKEN }}"
-      - id: "get_version"
+      - name: "create dist dir"
+        run: |
+          mkdir dist
         uses: "battila7/get-version-action@v2"
-      - uses: actions/download-artifact@v2
+      - uses: "actions/download-artifact@v2"
         with:
-          name: darwin_zed
-          path: zed_${{ steps.get_version.outputs.version-without-v }}_darwin_amd64.tar.gz
+          name: "darwin_zed"
+          path: "dist/zed_${{ steps.get_version.outputs.version-without-v }}_darwin_amd64.tar.gz"
```

Comment:
why not use $TEMPDIR?

Reply:
the archive needs to be moved into a place that's gitignored, or goreleaser will refuse to run (dist is gitignored and is the directory goreleaser puts its output archives too)

Reply:
see: https://github.com/authzed/zed/runs/3739410030?check_suite_focus=true